### ISSUE TITLE
[Question] Dynamic Shopify API Client

### ISSUE THREAD

**Issue by fabiensebban** [stephenbaidu/mojofz-app#1]
Hello,

I found your project looking for a way to set dynamically the Shopify client before request the Shopify GraphQL API.
I saw that you init the Shopify client in the [shopify init file.](https://github.com/stephenbaidu/mojofz-app/blob/master/config/initializers/shopify.rb)

How would you do to init a dynamic Shopify Client depending on the Shopify private app my app wants to query?

In my case, I need to sync multiple store information. To do this, I want to get information from a store and sent it to another store in the same execution.

**Comment by stephenbaidu** [comment]
@fabiensebban I don't think it is supported. You might have to do this via partner accounts. You probably found [this link](https://community.shopify.com/c/Shopify-APIs-SDKs/Accessing-Multiple-Stores-from-ShopifyAPI/m-p/301995/highlight/true#M14793) already but do check it out. And by the way, I'm still new to their API and this repo was for an interview task 😄

**Comment by fabiensebban** [comment]
@stephenbaidu , thank you for your answer!
I found this to dynamically set the query constant : 

https://stackoverflow.com/questions/50382830/graphqlclientdynamicqueryerror-expected-definition-to-be-assigned-to-a-stati/51681530?stw=2#51681530

It looks a little bit dirty but it works. Do you think it can affect my server performance to use the ruby Kernel function?
